admin-manual/installation/linux/ubuntu-bionic.rst

```diff
@@ -242,64 +217,19 @@ in :file:`/etc/nginx/sites-available/atom`.
 
    }
 
-Now you need to restart Nginx:
-
-.. code-block:: bash
-
-   sudo service nginx restart
-
-.. _linux-ubuntu-trusty-dependency-httpd-apache:
-
-Apache
-``````
-.. warning::
-
-   Remember that our preferred choice is
-   :ref:`linux-ubuntu-trusty-dependency-httpd-nginx` but it is perfectly
-   possible to use Apache and we have verified that it works.
-
-Install the necessary packages:
-
-.. code-block:: bash
-
-   sudo apt-get install apache2 libapache2-mod-xsendfile
-
-Enable the required modules:
+Now you need to enable and reload Nginx:
 
 .. code-block:: bash
 
-   sudo a2enmod rewrite xsendfile
-
-The configuration of your virtual server shoud look like the following:
-
-.. code-block:: apache
+   sudo systemctl enable nginx
+   sudo systemctl reload nginx
 
-    <VirtualHost *:80>
-      DocumentRoot /var/www/atom
-      RewriteEngine On
-      RewriteRule ^/uploads/r/([^/]*)/conf/(.*)$ /var/www/atom/uploads/r/$1/conf/$2 [L]
-      RewriteRule ^/uploads/(.*)$ /var/www/atom/uploads/$1 [L]
-      <LocationMatch ^/uploads>
-        XSendFile On
-        XSendFilePath /var/www/atom/uploads
-        SetEnv ATOM_XSENDFILE 1
-      </LocationMatch>
-    </VirtualHost>
-
-You also need to decide if you are going to use php5-fpm or mod_php. We prefer
-the former, in combination with Nginx, but you can combine php5-fpm and Apache
-as long as you install the Apache module mod_fastcgi (the corresponding Ubuntu
-package is called libapache2-mod-fastcgi). We have not tried this ourselves but
-it is definitely `possible <https://wiki.apache.org/httpd/PHP-FPM>`__.
-
-.. _linux-ubuntu-trusty-dependency-php:
```

Comment:
I think it's okay to remove the Apache section, I'd also remove it from the Xenial docs for consistency.

Reply:
The trusty file was removed, I don't know why git thinks that it was renamed to bionic...

The apache install instructions are not part of the xenial or bionic installs, am I missing something?

Reply:
I thought it would be also in the Xenial docs, thanks!